conf/pcls.routes

```diff
@@ -24,7 +24,7 @@ GET        /:srn/change-check-answers-pcls/:index
 POST       /:srn/change-check-answers-pcls/:index                                                       controllers.nonsipp.memberreceivedpcls.PclsCYAController.onSubmit(srn: Srn, index: Max300, mode: Mode = CheckMode)
 POST       /:srn/change-submit-check-answers-pcls/:index                                                controllers.nonsipp.memberreceivedpcls.PclsCYAController.onSubmit(srn: Srn, index: Max300, mode: Mode = ViewOnlyMode)
 GET        /:srn/check-answers-pcls/:index/view/:year/:current/:previous                                controllers.nonsipp.memberreceivedpcls.PclsCYAController.onPageLoadViewOnly(srn: Srn, index: Max300, mode: Mode = ViewOnlyMode, year: String, current: Int, previous: Int)
-GET        /:srn/submit-check-answers-pcls/view/:year/:current/:previous                                controllers.nonsipp.memberreceivedpcls.PclsCYAController.onSubmitViewOnly(srn: Srn, page: Int, year: String, current: Int, previous: Int)
+GET         /:srn/submit-check-answers-pcls/view/:page/previous/:year/:current/:previous                controllers.nonsipp.memberreceivedpcls.PclsCYAController.onSubmitViewOnly(srn: Srn, page: Int, year: String, current: Int, previous: Int)
```

Comment:
Is this needed or accidental? As the ticket is about transfers in

Reply:
This one I added @kberec , because from CYA page navigation is not working previously, now it's fine